test/pom.xml

```diff
@@ -36,4 +36,8 @@
         <module>test-integration</module>
     </modules>
 
-</project>
\ No newline at end of file
+    <properties>
+        <maven.javadoc.skip>true</maven.javadoc.skip>
```

Comment:
Whats the usage of this property

Reply:
It's for "javadoc: error - No source files for package org.wso2am.micro.gw.mockbackend
solution: exclude test package from javadoc"